org.eclipse.scanning.test/src/org/eclipse/scanning/test/points/GridTest.java

```diff
@@ -423,9 +422,10 @@ public void testFillingBoundingCircleSkewed() throws Exception {
 				new Point(2000, 0.4125, 2000, 15.375),
 				new Point(3100, 1.1325, 3100, 16.425) };
 		for (Point p : points) {
-			ArrayList<Collection<String>> names = new ArrayList<Collection<String>>();;
+			ArrayList<Collection<String>> names = new ArrayList<Collection<String>>();
 			names.add(Arrays.asList(new String[] {"x", "y"}));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)